backend/tests/unit/graphql/auth/query_permission_checker/test_object_permission_checker.py

```diff
@@ -7,11 +7,7 @@
 
 from infrahub.auth import AccountSession, AuthType
 from infrahub.core.account import ObjectPermission
-from infrahub.core.constants import (
-    InfrahubKind,
-    PermissionAction,
-    PermissionDecision,
-)
+from infrahub.core.constants import InfrahubKind, PermissionAction, PermissionDecision
```

Comment:
Nothing wrong here, more a question about the settings of your editor. I have a feeling that we have some different settings that could cause this change to happen?

Reply:
I think it's because I removed the last `,` after reverting a copy/paste or something. Trailing commas are a thing that can get the formatting all different quickly.